src/App.js

```diff
@@ -37,551 +54,226 @@ class App extends React.Component {
     caching: false,
   };
 
-  componentDidMount = () => {
-    // Create new intel instance
-    this.intel = new Intel();
-    // Create new session link for easy access
-    this.session = this.intel.snekclient.session;
-    // Begin session
-    this.begin();
-  };
-
-  //#region > Refetch Checking
-  /**
-   * Check for refetch for a specific username.
-   *
-   * @param {string} username The username associated with a profile page
-   * @returns {boolean} True if a refetch is required otherwise False
-   */
-  refetchRequired = (username) => {
-    const loading = this.state.loading;
-    const fetchedUser = this.state.fetchedUser;
-
-    if (!loading) {
-      if (!fetchedUser && fetchedUser !== false) {
-        return true;
-      } else if (
-        fetchedUser &&
-        !this.usernameMatchesFetchedUsername(username)
-      ) {
-        return true;
-      }
-      return false;
-    }
+  globalFunctions = {
+    /** Authentication Actions*/
```

Comment:
Since when do we use this type of commenting in the front?

Why not use `//`?

Reply:
I'm sorry. Might be my influence ^^